caffe2/ideep/operators/operator_fallback_ideep.h

```diff
@@ -121,13 +121,23 @@ class IDEEPFallbackOp final : public IDEEPOperator {
           "output type who needs copying.");
       const auto& src = local_output_blobs_[i]->template Get<TensorCPU>();
 
+      auto src_dims = src.dims();
```

Comment:
The src_dims on line 149 is shadowing this one. Can you fix that? Due to the compiler flags we use, it causes a build failure.